echidna-manifest.txt

```diff
@@ -1,4 +1,8 @@
 # Manifest file for Echidna (automatic publication).
 webdriver-spec.html?specStatus=WD;shortName=webdriver respec
 bug-assist.js
-graphics/
+graphics/note1actions.svg
+graphics/note2actions.svg
+graphics/note3actions.svg
+graphcis/note4actions.svg
```

Comment:
(FYI, the latest attempt to publish automatically [failed because of a typo here](https://lists.w3.org/Archives/Public/public-tr-notifications/2015Aug/0021.html) :)